pages/about.html

```diff
@@ -4,9 +4,51 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet" />
+  <link href="../styles/about.css" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Elaine Smith | About</title>
 </head>
 <body>
-  
+  <header>
+    <img src="/assets/elainee-logo.png" alt="elainee logo written in blue bubble font" id="main-logo" />
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="portfolio.html">Portfolio</a>
+      <!-- <a href="blog.html">Blog</a> -->
+      <a href="about.html">About</a>
+    </nav>
+  </header>
+  <main id="main-content">
+    <aside id="sidebar">
+      <img src="/assets/elaine-profile.jpeg" alt="profile photo" id="profile-pic" />
+      <h3>My Story</h3>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)